tale/dungeon/DungeonEntrance.py

```diff
@@ -0,0 +1,48 @@
+from tale.base import Exit
+from tale.coord import Coord
+from tale.dungeon.dungeon import Dungeon
+from tale.dungeon.dungeon_generator import ItemPopulator, LayoutGenerator, MobPopulator
+from tale.llm.dynamic_story import DynamicStory
+from tale.zone import Zone
+
+
+class DungeonEntrance(Exit):
+    """
+    A special exit that leads to a dungeon.
+
+    This can be added to any normal location to provide access to a dungeon.
+    """
+
+    def build_dungeon(self, story: 'DynamicStory', llm_util) -> Dungeon:
```

Comment:
Please create a DungeonConfig class, where name, description, items and races can be defined